### PR TITLE
Even more ViewSet instance attributes

### PR DESCRIPTION
**Pull request by intgr**
These attributes are not documented and were reverse engineered from DRF source code.

### CODE REVIEW

File: rest_framework-stubs/viewsets.pyi [hunk 1]
```diff
@@ -12,11 +12,18 @@ def _is_extra_action(attr: Any) -> bool: ...
...
 class ViewSetMixin(object):
+    # Classvars assigned in as_view()
     name: Optional[str]
     description: Optional[str]
     suffix: Optional[str]
     detail: bool
     basename: str
+    # Instance attributes assigned in view wrapper
+    action_map: Dict[str, str]
+    args: List[Any]
```
Comment (sobolevn) [inline rest_framework-stubs/viewsets.pyi]: Its't it `Tuple[Any, ...]`?

Comment (intgr) [reply rest_framework-stubs/viewsets.pyi]: Oh yes, you are right. Finally found the time to fix this.

**Comment by intgr** [comment]
Rebased my branch. I see that #54 already added the `action` attribute :)

**Comment by sobolevn** [comment]
Thanks again!

**Pull request merged**